tests/mbis-5/input.dat

```diff
@@ -22,6 +22,22 @@ octupoles_ref = psi4.Matrix.from_list([ #TEST
   -2.23989073e-12, -4.92237390e-07, -1.97235542e-07,  4.45294729e-01,
   -9.80994678e-08,  1.24298633e+00]])   #TEST
 
+# Reference Atomic Volumes from Horton
+# Zn 63.44244885428372
+# O  27.11773534802673
+
+avols_ref = psi4.Matrix.from_list([  #TEST
+ [63.44245356],    #TEST
+ [27.11771735]])   #TEST
```

Comment:
any ref data available to check hypervolume?

Reply:
This may not be necessary if <r^3> code works, and code is written properly.